addons/binding/org.openhab.binding.kvv/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="kvv"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- KVV Thing Type -->
+	<thing-type id="kvvstation">
+		<label>KVV Station</label>
+		<description>Sample thing for KVV Binding</description>
+
+		<channels>
+			<channel id="train0_name"        typeId="kvvstation-train-name" />
+			<channel id="train0_destination" typeId="kvvstation-train-destination" />
+			<channel id="train0_eta"         typeId="kvvstation-train-eta" />
+			<channel id="train1_name"        typeId="kvvstation-train-name" />
+			<channel id="train1_destination" typeId="kvvstation-train-destination" />
+			<channel id="train1_eta"         typeId="kvvstation-train-eta" />
```

Comment:
You could simplify this definition (and add some more structure to it) by using a [channel group](https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#channel-groups) for the information of each train.